fhirpipe/run.py

```diff
@@ -75,7 +75,12 @@ def run(
             # Transform
             fhir_instances = transformer.transform(chunk, resource_mapping, analysis)
             # Load
-            loader.load(fhirstore, fhir_instances, resource_mapping["definition"]["type"])
+            if not multiprocessing:
```

Comment:
I think that the shape of the return value of transformer.transform should be always the same (so we no longer need this condition here). What do you think ?

Reply:
Agreed!
But I also wanted to avoid to concatenate the lists on the transform side in case of multiprocessing, I don't how efficient Python is with that... But I'm certainly overthinking this ^^

Reply:
Changed to use a shared list